doc/procs/db_domain_exists.md

```diff
@@ -0,0 +1,26 @@
+qc::db_domain_exists
+===========
+
+part of [Database API](../db.md)
+
+Usage
+-----
+`db_domain_exists domain_name`
+
+Description
+-----------
+Checks if the given domain exists in the database.
```

Comment:
Would be a good idea. I don't think it's entirely clear what a domain is without the documentation.

Should the PostgreSQL domain docs be linked in all db_domain docs?